alephmarcreader/tests/test_MarcXMLReader.py

```diff
@@ -224,7 +224,13 @@ def test_get_bibliographic_references(self):
         self.assertEqual(doc_states[0].prefix, u'Druck')
         self.assertEqual(doc_states[0].reference, u'Joh. I B. Briefe 1, p.444')
 
+    def test_unexpected_cardinality(self):
```

Comment:
Does this test check for anything? I think you should remove it.

Reply:
It forces the situation, where this warning would be necessary. But it doesn't really test anything.  
Shall I remove it? In that case I also don't need the new file `wrong_cardinality.xml`.

Reply:
I think you can leave the test file where it is. It does no hurt.

Reply:
I decided to go the overkill way nonetheless.

Reply:
Awesome!

Reply:
let's not get overly excited, here! ;)

Reply:
did you find that on stackoverflow?

Reply:
jap: https://stackoverflow.com/questions/33767627/python-write-unittest-for-console-print

Reply:
please just add this link and a short explanation to the code

Reply:
aye